setup.py

```diff
@@ -11,7 +11,7 @@
 
 setup(
     name='remodel',
-    version='0.4.4',
+    version='0.4.5',
```

Comment:
I need to check what else needs/can be integrated in this version, so I would leave the version bump out of this pull request.